.github/workflows/build.yml

```
@@ -0,0 +1,30 @@
+    strategy:
+        matrix:
+            os: [windows-latest, ubuntu-latest, macOS-latest]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Fetch all branches and tags
+        run: git fetch --unshallow --prune
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+            dotnet-version: 9.0.x            
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)